workflows/prognostic_c48_run/tests/test_fv3fit_transformer.py

```diff
@@ -66,7 +71,45 @@ def add_one_to_temperature():
     regression_state(out, regtest)
 
 
-def test_multimodel_adapter(state, tmpdir_factory):
+def test_multimodel_adapter(state):
+    nz = 63
+    outputs = {
+        "Q1": np.full(nz, 1 / 86400),
+        "tendency_of_air_temperature_due_to_nudging": np.full(nz, 1 / 86400),
+        "implied_surface_precipitation_rate": 10 / 86400,
```

Comment:
pretty complex mocking logic here with hard-coded names. Suggests the underlying APIs are not adequately decoupled.

Reply:
new tests are a bit simpler